libgrive/src/json/Val.hh

```diff
@@ -138,6 +138,7 @@ template <> struct Val::Type2Enum<double>		{ static const TypeEnum type = double
 template <> struct Val::Type2Enum<std::string>	{ static const TypeEnum type = string_type ; } ;
 template <> struct Val::Type2Enum<Val::Array>	{ static const TypeEnum type = array_type ; } ;
 template <> struct Val::Type2Enum<Val::Object>	{ static const TypeEnum type = object_type ; } ;
+template <> struct Val::Type2Enum<std::vector<std::string> >	{ static const TypeEnum type = string_list_type ; } ;
```

Comment:
I don't like this thing... array of strings should be implemented via array Val of string Vals, like in json: [ str1, str2, ... ]